src/node.cc

```diff
@@ -117,6 +117,14 @@
 #include <string>
 #include <vector>
 
+// =========== windows headers ============
+
```

Comment:
The comment shouldn't be necessary given the presence of the `#ifdef _WIN32` block

```suggestion
```